### ISSUE TITLE
Non-zero exit code when running with --help

### ISSUE THREAD

**Issue by awvwgk** [qcscine/readuct#2]
Please don't raise non-zero exit code when the program finishes correctly after printing the help message.

```
❯ readuct --help
Recognized options:
  -h [ --help ]         Produce this help message
  -c [ --config ] arg   YAML input file to read

❯ echo $?
1
```

**Comment by weymutht** [comment]
Thanks! This will be fixed in the next release.

**Comment by reiher-research-group** [comment]
This is fixed in version 4.0.0 of ReaDuct which has been released today.
